doc/reference/reference_lua/box_info/election.rst

```diff
@@ -0,0 +1,39 @@
+.. _box_info_election:
+
+================================================================================
+box.info.election
+================================================================================
+
+.. module:: box.info
+
+.. data:: election
+
+   Show the current state of a replica set node in regards to
+   :ref:`leader election <repl_leader_elect>`, namely,
+   election state (mode), election term, vote in the current term,
+   and the leader ID of the current term.
+
+   **Example:**
+
+   .. code-block:: tarantoolsession
+
+      tarantool> box.info.election
+      ---
+      - state: follower
+        vote: 0
+        leader: 0
+        term: 1
+      ...
+
+   IDs in the ``box.info.election`` output are the replica IDs visible in
+   the ``box.info.id`` output on each node and in the ``_cluster`` space.
+
+   :ref:`State <repl_leader_elect_state>` can be ``leader``, ``follower``,
```

Comment:
WARNING: undefined label: repl_leader_elect_state (if the link has no caption the label must precede a section header)

Reply:
will be fixed later in https://github.com/tarantool/doc/issues/1773